### PR TITLE
Added support for profiles

### PR DESCRIPTION
**Pull request by ChillarAnand**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,15 @@ $ appknox files 4
 ----  ------  ---------  --------------
    6  MFVA            1               6
    7  MFVA            1               6
+
```
Comment (viren-nadkarni) [inline README.md]: Could add a comment here on what the command does?

Comment (viren-nadkarni) [inline README.md]: Perhaps we should remove these examples from here? Quick-start should be as simple as possible and it is unlikely normal users will have multiple accounts with Appknox and that they'll use profiles...

File: appknox/cli.py [hunk 2]
```diff
@@ -43,10 +47,32 @@ def table(model, instances, ignore=list()):
...
+def remove_profile(name):
+    result = config.remove_section(name)
+    with open(CONFIG_FILE, 'w') as fh:
+        config.write(fh)
+    return result
+
+
 @click.group()
 @click.option('-v', '--verbose', count=True, help='Specify log verbosity.')
+@click.option('-n', '--profile', default=DEFAULT_PROFILE)
```
Comment (viren-nadkarni) [inline appknox/cli.py]: why not `-p`?

Comment (ChillarAnand) [inline appknox/cli.py]: Won't it be ambiguous with `-p` password?

**Pull request merged**